dotcom-rendering/src/amp/components/Elements.tsx

```diff
@@ -109,8 +108,7 @@ export const Elements = (
 	isImmersive: boolean,
 	adTargeting?: AdTargeting,
 ): JSX.Element[] => {
-	const cleanedElements = enhance(elements);
```

Comment:
I've pulled this up into AMPs `index.tsx`, the new minifer library uses async which is hard to use in Components.